packages/react/src/components/Slider/Slider.test.tsx

```diff
@@ -34,95 +25,94 @@ describe('Slider', () => {
   });
 
   it('renders correctly', () => {
-    const component = create(<Slider label="I am a slider" />);
-    const tree = component.toJSON();
-    expect(tree).toMatchSnapshot();
+    const { container } = render(<Slider label="I am a slider" />);
+    expect(container).toMatchSnapshot();
   });
 
   it('renders range slider correctly', () => {
-    const component = create(<Slider label="I am a ranged slider" ranged defaultValue={5} />);
-    const tree = component.toJSON();
-    expect(tree).toMatchSnapshot();
+    const { container } = render(<Slider label="I am a ranged slider" ranged defaultValue={5} />);
+    expect(container).toMatchSnapshot();
   });
 
   it('can set aria-labelledby attribute', () => {
-    wrapper = mount(<Slider aria-labelledby="custom-label" />);
-    const sliderSlideBox = wrapper.find('.ms-Slider-slideBox');
-    expect(sliderSlideBox.getDOMNode().getAttribute('aria-labelledby')).toEqual('custom-label');
+    const { getByRole } = render(<Slider aria-labelledby="custom-label" />);
+    expect(getByRole('slider').getAttribute('aria-labelledby')).toBe('custom-label');
   });
 
   it('can provide the current value', () => {
     const slider = React.createRef<ISlider>();
 
-    wrapper = mount(<Slider defaultValue={12} min={0} max={100} componentRef={slider} />);
+    render(<Slider defaultValue={12} min={0} max={100} componentRef={slider} />);
     expect(slider.current?.value).toEqual(12);
   });
 
   it('can provide the current range', () => {
     const slider = React.createRef<ISlider>();
 
-    wrapper = mount(<Slider defaultValue={12} min={0} max={100} componentRef={slider} ranged />);
+    render(<Slider defaultValue={12} min={0} max={100} componentRef={slider} ranged />);
     expect(slider.current?.range).toEqual([0, 12]);
   });
 
   it('can set id', () => {
-    wrapper = mount(<Slider id="test_id" styles={{ titleLabel: 'test_label' }} />);
+    const { container, getByRole } = render(<Slider id="test_id" styles={{ titleLabel: 'test_label' }} />);
 
-    const sliderSlideBox = wrapper.find('.ms-Slider-slideBox');
-    expect(sliderSlideBox.getDOMNode().id).toEqual('test_id');
+    expect(getByRole('slider').id).toEqual('test_id');
 
     // properly associates label with custom id
-    const label = wrapper.find('label.test_label');
-    expect(label.prop('htmlFor')).toBe('test_id');
+    const label = container.getElementsByClassName('test_label')[0];
```

Comment:
I think it might be slightly better/more idiomatic to get the label by role, then test its class and for. Something like this (haven't run this in TS and might be remembering property names incorrectly).
```ts
const label = getByRole('label');
expect(label.className).toMatch(/test_label/);
expect(label.getAttribute('for')).toBe('test_id');
```

And similar in the next test.

Reply:
Yup that's what I initially had in mind but apparently the label element doesn't have a corresponding aria role
![image](https://user-images.githubusercontent.com/8649804/156455169-6825a124-9191-4360-933d-bfb99085ea38.png)

Reply:
Oh interesting...I didn't see any other suggestions from a brief online search, so I guess we can stick with this.